importexport/inc/class.importexport_import_ui.inc.php

```diff
@@ -74,7 +74,9 @@ public function import_dialog($content = array()) {
 					$plugin = new $definition_obj->plugin;
 
 					// Check file encoding matches import
-					$sample = file_get_contents($content['file']['tmp_name'],false, null, 0, 1024);
+					$sample = file_get_contents($content['file']['tmp_name']);
+					$sample = mb_substr( $sample, 1024 );
+
```

Comment:
Ok, I will keep that in mind.

Yes, that's better than fetching the whole file. I fixed it with another commit